bom/pom.xml

```diff
@@ -485,6 +485,18 @@
                 <version>${mockito.version}</version>
                 <scope>import</scope>
             </dependency>
+            <dependency>
```

Comment:
The camunda libs should not be managed here as they are specific to the java-testing plugin and will not be re-used by other jQA components or plugins